examples/Spinners/LoadSpinner.md

```diff
@@ -0,0 +1,13 @@
+A load spinner component that can be used to show the loading state of the data.
+```jsx
+<LoadSpinner>
+  Loading Data
+</LoadSpinner>
+```
+
+To style the text to have a dark light color.
+```jsx
+<LoadSpinner color='light'>
+  Loading Data
+</LoadSpinner>
+```
```

Comment:
Here we want to use the actual enum value (e.g. `SPINNER_TEXT.LIGHT`) instead of the literal string `'light'`. It's effectively doing the same thing, but that's how we'd want these values to actually be used.

I would also recommend wrapping this spinner in a `div` with a dark background, as a way to show the different context in which is would be used.

````suggestion
```tsx
import { useContext } from 'react';
import { ThemeContext } from 'styled-components';
import { SPINNER_TEXT } from 'mark-one';

const theme: BaseTheme = useContext(ThemeContext);

<div style={{background: theme.color.background.darker}}>
  <LoadSpinner color={SPINNER_TEXT.LIGHT}>
    Loading Data
  </LoadSpinner>
</div>
```
````